corehq/apps/hqwebapp/models.py

```diff
@@ -361,6 +361,12 @@ def is_viewable(self):
                 return domain_has_apps(self.domain)
         return False
 
+    @property
+    @memoized
+    def url(self):
+        from corehq.apps.dashboard.views import default_dashboard_url
+        return default_dashboard_url(self._request, self.domain)
```

Comment:
This displays `/a/domain/dashboard/project` instead of `a/domain/dashboard/` which redirects to the former